code-samples/messaging/send-mms.php

```diff
@@ -0,0 +1,34 @@
+<?php
+	/*You get the environment parameters from your 

```

Comment:
Not sure what is going on here, but this indentation is awkward. Have code sample begin flush left, and indent from there.

Reply:
@byrnereese can you explain what you mean by this?